core/src/main/res/values-night/color.xml

```diff
@@ -5,16 +5,15 @@
   <!--The same color values are defined for dark theme, but with values more appropriate for -->
   <!--low-luminance UIs. These colors will be used by night/themes.xml.-->
 
-  <color name="color_primary">@color/blue400</color>
```

Comment:
Ideally this file should only contain colors which are only used in night mode and not perform overriding of any other color.

Or this file simply should not exist but I think the former may be more correct from a linting perspective